docs/spec.md

```diff
@@ -2828,6 +2829,42 @@ behavior is undefined. More formally, for all `id < jd` from `indices(result)`,
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.get_dimension_size
+
+### Semantics
+
+Returns the size of the given `dimension` of the `operand`.
+
+### Inputs
+
+| Name          | Type                         |
+|---------------|------------------------------|
+| `operand`     | tensor of any supported type |
+| `dimension`   | constant of type `si64`      |
+
+### Outputs
+
+| Name     | Type                                |
+|----------|-------------------------------------|
+| `result` | 0-dimensional tensor of type `si32` |
+
+### Constraints
+
+  * (C1) 0 $\le$ `dimension` $\lt$ rank of `operand`. [todo](https://github.com/openxla/stablehlo/issues/790)
```

Comment:
\`rank(operand)\`, similarly to other places in the spec.

Reply:
Done. Sent https://github.com/openxla/stablehlo/pull/815 for a few existing places.